Sources/WebAuthn/WebAuthnError.swift

```diff
@@ -12,57 +12,120 @@
 //
 //===----------------------------------------------------------------------===//
 
-public enum WebAuthnError: Error, Equatable {
+public struct WebAuthnError: Error, Hashable {
+    var reason: Reason
```

Comment:
Does this need to be mutable? Might be better to be immutable and open it up if needed

Reply:
Nope, but it was internal, so I figured what's the harm haha. Changed.